requirements.txt

```diff
@@ -1,5 +1,6 @@
-argparse==1.2.1
-cchardet
+six
+argparse
+cchardet==1.1.2
```

Comment:
I think i did that because 1.1.2 was the first release that supported 3.6 officially. If someone has a previous version installed and doesn't run `pip install` with the upgrade flag I wonder if that will potentially cause an issue.